platforms/shared/tests/test-client/main.cpp

```diff
@@ -23,6 +23,8 @@ using namespace cv;
 
 namespace prc = prism::connect;
 
+typedef boost::chrono::system_clock::time_point time_point;
```

Comment:
This code including typedef was there for quite a while. Choosing between _t suffix and TimePoint, I'd choose time_point_t, because TimePoint looks like class name, rather than a value type.